spaces/S000125/properties/P000065.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000125
+property: P000065
+value: true
+refs:
+- doi: 10.1007/978-1-4612-6290-9_6
+  name: Counterexamples in Topology
+---
+
+The space contains a line segment and is contained within $\mathbb R^2$.
```

Comment:
The space does not contain a line segment, but does contain all the points with irrational $y$-coordinate within a slanting line segment.

Reply:
fixed in 210a8e7